### ISSUE TITLE
request: static binary for linux

### ISSUE THREAD

**Issue by leleleSDX** [nicoboss/nsz#146]
please for the love of god. steamos keeps breaking distrobox. i'll name my firstborn after you.

**Comment by nicoboss** [comment]
I agree that binary releases for Linux would make sense. One of the reasons there isn't is because Python comes preinstalled with most Linux distributions and pip can be installed using `apt install python3-pip` after which `pip install nsz[gui]` can be used. macOS users want their own binary builds as well so with adding binary builds for macOS there also should be binary builds added for Linux.

Are you running Linux on AMD64, ARM64 or RISCV64? AMD64, ARM64 and RISCV64 all require their separate binary releases. Having to create, test and release a sperate binary build for every OS and Archidecture combination is a lot of work and the main reason there are currently only binary builds for Windows AMD64 but this will hopefully change with the next release but I can't promise anything.

**Comment by jpyper** [comment]
You could always use [pyinstaller](https://github.com/pyinstaller/pyinstaller) to make a stand-alone executable from the project source tree or release tarball. It works a treat on Arch Linux amd64 on my main PC and Arch Linux ARM aarch64 on the Raspberry Pi 4. YMMV.

**Comment by Brycey92** [comment]
Hello @jpyper please try the latest version from the [releases page](https://github.com/nicoboss/nsz/releases).
